test/verify/check-docker

```diff
@@ -498,5 +498,68 @@ CMD ["/bin/sh"]
         # This should be updated
         b.wait_in_text("#container-details-cpu-row", "1024 shares")
 
+class TestAtomicScan(MachineCase):
+    def testBasic(self):
+        b = self.browser
+        m = self.machine
+
+        m.execute("systemctl start docker")
+
+        # HACK put scan results for one of the images into /var/lib/atomic
```

Comment:
Is there a bug/link to go with this `HACK`? So we know when we can change this / remove the hack.